plugins/optimization-detective/load.php

```diff
@@ -27,19 +27,22 @@
 
 define( 'OPTIMIZATION_DETECTIVE_VERSION', '0.1.0' );
 
-require_once __DIR__ . '/hooks.php';
-
 // Storage logic.
 require_once __DIR__ . '/class-od-data-validation-exception.php';
 require_once __DIR__ . '/class-od-url-metric.php';
 require_once __DIR__ . '/class-od-url-metrics-group.php';
 require_once __DIR__ . '/class-od-url-metrics-group-collection.php';
 require_once __DIR__ . '/class-od-storage-lock.php';
```

Comment:
I guess all of these should go into `storage` then too.

Reply:
Moved in b5623fd0

Reply:
@westonruter Maybe. `OD_Storage_Lock` probably yes, for the URL metric classes not necessarily - though I don't feel strongly about it. Those are the central infrastructure classes used for the entire functionality, IMO they aren't _just_ storage related as e.g. the post type class is.

I'm okay with either, though please align the test files structure with whatever you go with here as well (e.g. right now the post type class tests are not in the `storage` folder even though that class is in the `storage` folder. Nit-picking... :)